driver/terraform.go

```diff
@@ -491,6 +504,22 @@ func (tf *Terraform) initTaskTemplate() error {
 	return nil
 }
 
+func (tf *Terraform) validateTask(ctx context.Context) error {
+	taskName := tf.task.Name()
+
+	if err := tf.init(ctx); err != nil {
```

Comment:
I was thinking about that too! I ended up leaving them in because the init method first checks if the workspace is already initialized, and I thought it would be better to not have applyTask() and inspectTask() depend on validateTask() running first. Maybe another point for moving the validate to be part of InitTask, though!

Reply:
Mmmm good point about not having a applyTask() and inspectTask() not depending on validateTask() running first. Agree that if validate gets moved in to InitTask, then it would be worth considering removing the terraform-init from apply and inspect!